hawc/apps/assessment/forms.py

```diff
@@ -44,6 +46,10 @@ class Meta:
         model = models.Assessment
         widgets = {
             "public_on": DateCheckboxInput,
+            "project_manager": autocomplete.ModelSelect2Multiple(
```

Comment:
We'll probably want to build a helper method or class to specify the bootstrap theme by default. But that may come when we start implementing